src/NuGet.Core/NuGet.Commands/NuGet.CommandLine/SourcesListFormat.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
The directory structure should be different: 

src/NuGet.Core/NuGet.Commands/NuGet.CommandLine/SourcesListFormat.cs  => 
src/NuGet.Core/NuGet.Commands/SourcesCommand/SourcesListFormat.cs 

See https://github.com/NuGet/NuGet.Client/tree/dev/src/NuGet.Core/NuGet.Commands

Reply:
fixing